recipes/biogeme/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "biogeme" %}
+{% set version = "3.2.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/biogeme-{{ version }}.tar.gz
+  sha256: fe1169e09fcebe6c100e3c887c015301cba1e6abcd0f546eaf6e949104d52847
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=63.4.1
+    - wheel
+    - pip
+  run:
+    - python >=3.7
+    - scipy >=1.14.0, <2
+    - matplotlib-base >=3.9.0, <4
+    - pandas >=2.2.2, <3
+    - tqdm >=4.66.4
+    - tomlkit >=0.12.5
+    - python-levenshtein >=0.25.1
+    - fuzzywuzzy >=0.18.0
+    - cythonbiogeme ==1.0.4
+    - biogeme_optimization ==0.0.10
+    - ipython >=8.25.0
```

Comment:
This is only found in biogeme examples. I would remove it here, as it really isn't a dependency.